bench/workgen/workgen.cxx

```diff
@@ -902,9 +956,26 @@ int ThreadRunner::op_run(Operation *op) {
     if (op->_transaction != NULL) {
         if (ret != 0 || op->_transaction->_rollback)
             WT_TRET(_session->rollback_transaction(_session, NULL));
-        else if (_in_transaction)
-            ret = _session->commit_transaction(_session,
-              op->_transaction->_commit_config.c_str());
+        else if (_in_transaction) {
+            // Set prepare, commit and durable timestamp if prepare is set.
+            if (op->_transaction->prepare_time > 0) {
+                time_us = WorkgenTimeStamp::get_timestamp();
```

Comment:
It looks like `prepare_time` and `commit_with_timestamp` should not both be set (if they are both set, then `commit_with_timestamp` would be ignored).  Given that, we should check these variables before running and issue an error if both are set.

Reply:
Added a condition to throw exception if `use_prepare_timestamp` and `use_commit_timestamp` is set.